apps/portals/src/configurations/adknowledgeportal/routesConfig.ts

```diff
@@ -74,6 +74,7 @@ const routes: GenericRoute[] = [
                     selectFacetColumnValue: 'WGS_Harmonization',
                     detailsPagePath:
                       '/Explore/Studies/DetailsPage?Study=syn22264775',
+                    unitDescription: 'Files',
```

Comment:
Determines the plot legend table column header: 
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/78d8ee75-576a-42cf-9d5c-faa8ed5dfcc6)